content/docs/start/bitbucket.md

```diff
@@ -13,19 +13,16 @@ all the supported CI systems.
    However, to follow along the commands, we recommend cloning your fork to your
    local workstation:
 
-   ```bash
-   git clone https://bitbucket.org/<your-username>/example-cml
+   ```cli
+   $ git clone https://bitbucket.org/<your-username>/example-cml
    ```
 
 2. ⚠️ Follow
-   [these instructions](https://cml.dev/doc/self-hosted-runners?tab=Bitbucket#personal-access-token)
-   to configure a Bitbucket token for CML.
-3. ⚠️ Follow
    [these instructions](https://cml.dev/doc/ref/send-comment#bitbucket) to
    enable the Pull Request Commit Links application.
 
-4. To create a CML workflow, copy the following into a new file on your main
-   branch, `bitbucket-pipelines.yml`:
+3. To create a CML workflow, copy the following into a new file,
```

Comment:
maybe people are better at bitbucket than I, but if you explicitly follow these instructions then `bitbucket-pipelines.yml` will be part of the PR (and not run)

Reply:
hence the explicit mention of the "main" branch

Reply:
github doesnt, workflows only need to be in the "main" branch for `workflow_dispatch` and `schedule` push and pr event are fine. not sure about gitlab